README.md

```diff
@@ -323,10 +366,7 @@ page.uploadFile('#selector', '/path/to/file').then(function() {
 });
 ```
 
-## Advanced
-Methods below are for advanced users. Most people won't need these methods.
```

Comment:
You would think that most people know about phantomjs and node. Unfortunately, that's not true :( I have found that majority of opened issues are about the lack node or phantom documentation. I had a few people using `invokeAsyncMethod` instead of actually the real method. 

Let's add the warning back. I am fine with not making it h3. I prefer an info box but I don't think that exists.